tests/System.IO.Compression.Tests/BrotliStreamTests.cs

```diff
@@ -14,6 +14,20 @@ public class BrotliStreamTests
     {
         static string brTestFile(string fileName) => Path.Combine("BrotliTestData", fileName);
 
+
+        [Theory(Skip = "Fails in VS - System.BadImageFormatException : An attempt was made to load a program with an incorrect format.")]
+        [InlineData(10)]
```

Comment:
If we are only testing sec = 10, there is no need for InlineData, just set sec = 10 in the test body and make this test a Fact.